tests/test_diffing/test_diff_writer.py

```diff
@@ -368,3 +369,74 @@ def test_dump_diff__returns_joined_list(self):
         result = self.writer.dump_diff(self.diff.config_diff)
         expected = '\n'.join(self.diff.config_diff)
         assert result == expected
+
+
+class TestColouredDiffLibWriter:
+    def setup_method(self, method):
+        self.stack_name = 'stack'
+
+        self.is_deployed = True
+        self.output_format = 'yaml'
+
+        self.output_stream = StringIO()
+
+        self.config1 = StackConfiguration(
+            stack_name=self.stack_name,
+            parameters={},
+            stack_tags={},
+            notifications=[],
+            role_arn=None
+        )
+
+        self.template1 = "foo"
+
+    @property
+    def template_diff(self):
+        return [
+            '--- file1.txt   2018-01-11 10:39:38.237464052 +0000\n',
+            '+++ file2.txt   2018-01-11 10:40:00.323423021 +0000\n',
+            '@@ -1,4 +1,4 @@\n',
+            ' cat\n',
+            '-mv\n',
+            '-comm\n',
+            ' cp\n',
+            '+diff\n',
+            '+comm\n'
+        ]
+
+    @property
+    def config_diff(self):
+        return []
+
+    @property
+    def diff(self):
+        return StackDiff(
+            self.stack_name,
+            self.template_diff,
+            self.config_diff,
+            self.is_deployed,
+            self.config1,
+            self.template1
+        )
+
+    @property
+    def writer(self):
+        return ColouredDiffLibWriter(
+            self.diff,
+            self.output_stream,
+            self.output_format
+        )
+
+    def test_plus_lines_are_coloured(self):
+        coloured = (
+            '\x1b[31m--- file1.txt   2018-01-11 10:39:38.237464052 +0000\n\x1b[39m\n'
```

Comment:
To make these tests more readable, I'd say you should use f-strings with the more readable colorama constants.

Like...
```suggestion
            f'{Fore.RED}--- file1.txt   2018-01-11 10:39:38.237464052 +0000\n{Fore.RESET}'
```

Reply:
In its current form, the expected lines here are pretty hard to read. And if this test started failing in the future, it'd be pretty hard for a maintainer to figure out what exactly SHOULD be going on here, since those arcane terminal escape codes are far from common knowledge.

Reply:
Arguably, there is redundancy in having `Fore.RED` etc in the code and their literal representation in the tests and it helps schmucks like me understand what a "Fore.RED" _really_ is! But I've changed it all the same.

(I'm loving the new quote style by the way!)